app/models/registration/EuTaxRegistration.scala

```diff
@@ -61,9 +65,37 @@ object RegistrationWithFixedEstablishment {
     Json.format[RegistrationWithFixedEstablishment]
 }
 
-final case class RegistrationWithoutFixedEstablishment(country: Country) extends EuTaxRegistration
+
+final case class RegistrationSendingGoods(
```

Comment:
To make this neater, I'm wondering whether the new tradingName / address can be added as an optional object under RegistrationWithoutFixedEstablishment? Simplifies the number of models, and in keeping with RegistrationWithFixedEstablishment?